docs/plugin_developers_guide/01_Fledge_plugins.rst

```diff
@@ -41,11 +41,136 @@ Plugins
+| plugin_eval            | Notification    | Takes JSON asset document to eval; Returns True if should "notify"                                    |
++------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
+| plugin_triggers        | Notification    | Returns JSON asset document describing what notification triggers have fired                          |
++------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
+| plugin_reason          | Notification    | Takes JSON asset document describing why notifications have fired                                     |
++------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
+| plugin_deliver         | Notification    | Takes name/notification/trigger/message strings to be sent to                                         |
+|                        | |br| delivery   | [notification delivery](#notification-delivery-types) channel.                                        |
++------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
+| plugin_send            | North           | Provides data,input_ref to be sent to North plugin target                                             |
++------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
+| plugin_poll            | South           | Initiates pull (return) of next set of data from south data source.                                   |
+|                        |                 | Only applicable for "poll" mode South plugins.                                                        |
++------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
+| plugin_start           | South           | Starts components needed to service async mode South operation, such as service threads.              |
++------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
+| plugin_register_ingest | South           | Registers callback and ingest "ref" which receive new data as available                               |
++------------------------+-----------------+-------------------------------------------------------------------------------------------------------+
+
+Notification delivery types
+---------------------------
+Notifications can be delivered through a variety of media.
+
+Notification delivery types include:
```

Comment:
There is a generated list that's is always up to date, better to reference his than hard code a list that will become out of date very quickly.